go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/Netflix/go-expect v0.0.0-20220104043353-73e0943537d2
 	github.com/PaesslerAG/jsonpath v0.1.1
-	github.com/aws/aws-sdk-go-v2 v1.22.2
-	github.com/aws/aws-sdk-go-v2/config v1.24.0
-	github.com/aws/aws-sdk-go-v2/credentials v1.15.2
-	github.com/aws/aws-sdk-go-v2/service/kms v1.26.1
+	github.com/aws/aws-sdk-go-v2 v1.22.1
```

Comment:
bad merge? the versions are rolling back

Reply:
yes, thanks for catching this, did a rebase as multiple changes were incorrect in the go.mod